docs/src/contributing_websocket_http_api.md

```diff
@@ -754,6 +754,8 @@ Search types are `All, Comments, Posts, Communities, Users, Url`
     site: Option<SiteView>,
     admins: Vec<UserView>,
     banned: Vec<UserView>,
+    online: usize,
```

Comment:
This is still broken right? Should mention that here.

Reply:
Done.